etc/kayobe/ansible/ovn-fix-chassis-priorities.yml

```diff
@@ -46,28 +49,31 @@
     ha_chassis_max_priority: 32767
     gateway_chassis_max_priority: "{{ ovn_nb_db_hosts_sorted | length }}"
   tasks:
-    - name: Fix ha_chassis priorities
-      ansible.builtin.command: >-
-        docker exec ovn_nb_db
-        bash -c '
-        ovn-nbctl find ha_chassis chassis_name={{ item }} |
-        awk '\''$1 == "_uuid" { print $3 }'\'' |
-        while read uuid; do ovn-nbctl set ha_chassis $uuid priority={{ priority }}; done'
-      loop: "{{ ovn_nb_db_hosts_sorted }}"
-      vars:
-        priority: "{{ ha_chassis_max_priority | int - ovn_nb_db_hosts_sorted.index(item) }}"
-      register: ha_chassis_command
-      changed_when: ha_chassis_command.rc == 0
+    - name: Fix OVN chassis priorities
+      when: kolla_enable_ovn
```

Comment:
nit: No need for a block here since the ovn_nb_leader group won't exist if OVN is disabled.

Reply:
done